hooks/pre/10-reset_data.rb

```diff
@@ -62,14 +41,13 @@ def empty_candlepin_database
 end
 
 def reset_candlepin
-  Kafo::KafoConfigure.logger.info 'Dropping Candlepin database!'
```

Comment:
You drop this line, but keep it in Pulp. Feels inconsistent

Reply:
It's not dropped, it's moved to `empty_db_in_postgresql`. Pulp2 doesn't use that function so I kept it there.

Reply:
Right. Makes sense. You could keep things the same by moving it in `empty_mongo` as well.